manifests/klusterlet/management/klusterlet-registration-deployment.yaml

```diff
@@ -70,7 +70,7 @@ spec:
           readOnly: true
         - name: hub-kubeconfig
           mountPath: "/spoke/hub-kubeconfig"
-        {{if eq .InstallMode "Detached"}}
+        {{if or (eq .InstallMode "Detached") (eq .InstallMode "Hosted")}}
```

Comment:
you would not need this or since the mode is always hosted, right?

Reply:
Fixed.